player/command.c

```diff
@@ -7815,6 +7815,9 @@ void mp_option_change_callback(void *ctx, struct m_config_option *co, int flags,
         && mpctx->vo_chain->is_sparse && !mpctx->ao_chain
         && mpctx->video_status == STATUS_DRAINING)
         mpctx->time_frame = opts->image_display_duration;
+
+    if (flags & UPDATE_DEMUXER)
+        mpctx->demuxer_changed = true;
```

Comment:
Useless cosmetic thing but this feels like it should be up where most of the other `flags & UPDATE_THING` stuff is.

Reply:
I placed it earlier initially but then noticed that `UPDATE_HWDEC` and `UPDATE_DVB_PROG` were already separated from other flags. I'm just gonna move those too.